crates/sui-graphql-rpc/src/types/move_object.rs

```diff
@@ -34,9 +37,17 @@ impl MoveObject {
         Some(MoveValue::new(type_, self.native.contents().into()))
     }
 
-    /// Determines whether a tx can transfer this object
-    async fn has_public_transfer(&self) -> Option<bool> {
-        Some(self.native.has_public_transfer())
+    /// Determines whether a transaction can transfer this object, using the TransferObjects
```

Comment:
do we need the "using the TransferObjects command" bit? I'd think that's the command someone adds to a PTB to actually transfer the object, which would be separate from someone checking whether an object is transferrable

Reply:
The distinction is that types do not have `store` can still be transferred if their module defines a custom transfer function, but I can also add a piece about `sui::transfer::public_transfer` which is the other place that cares about the `store` rule.